Search-and-Rescue/SearchAndRescue-FE
# Separate components

#### What does this PR do?

Updates the navigation system to include three new (separate) sub-navigators for contacts, vehicles and gear. All include new components for their respective list and individual item.

#### Where should the reviewer start?

Look at the navigator file and each new component. Each are copies of the CategoryList and ListItem.

#### How should this be manually tested?

Pull the branch to check in the simulator, and we will need to add snapshots, etc.

#### Any background context you want to provide?

N/A

#### What are the relevant tickets?

https://github.com/Search-and-Rescue/SearchAndRescue-FE/issues/45

#### Screenshots (if appropriate)

N/A

#### Questions:

Let's come back to this process if we decide to do the same w/Trips.
